lib/rules/font-weight-notation/index.js

```diff
@@ -98,73 +103,108 @@ const rule = (primary, secondaryOptions) => {
 				return;
 			}
 
-			if (
-				weightValue.toLowerCase() === INHERIT_KEYWORD ||
-				weightValue.toLowerCase() === INITIAL_KEYWORD
-			) {
+			if (includesOnlyFunction(weightValue)) {
+				return;
+			}
+
+			const lowerWeightValue = weightValue.toLowerCase();
+
+			if (lowerWeightValue === INHERIT_KEYWORD || lowerWeightValue === INITIAL_KEYWORD) {
 				return;
 			}
 
 			if (
 				optionsMatches(secondaryOptions, 'ignore', 'relative') &&
-				keywordSets.fontWeightRelativeKeywords.has(weightValue.toLowerCase())
+				keywordSets.fontWeightRelativeKeywords.has(lowerWeightValue)
 			) {
 				return;
 			}
 
-			const weightValueOffset = decl.value.indexOf(weightValue);
-
 			if (primary === 'numeric') {
 				const parent = decl.parent;
 
 				if (parent && isAtRule(parent) && parent.name.toLowerCase() === 'font-face') {
-					const weightValueNumbers = postcss.list.space(weightValue);
-
-					if (!weightValueNumbers.every((value) => isNumbery(value))) {
-						return complain(messages.expected('numeric'));
+					// @font-face allows multiple values.
+					for (const valueNode of getWeightValueNodes(weightValue)) {
+						if (!isNumbery(valueNode.value)) {
+							return complain(messages.expected('numeric'), valueNode);
+						}
 					}
 
 					return;
 				}
 
 				if (!isNumbery(weightValue)) {
-					return complain(messages.expected('numeric'));
+					return complain(messages.expected('numeric'), weightValueNode);
 				}
 			}
 
 			if (primary === 'named-where-possible') {
 				if (isNumbery(weightValue)) {
 					if (WEIGHTS_WITH_KEYWORD_EQUIVALENTS.has(weightValue)) {
-						complain(messages.expected('named'));
+						complain(messages.expected('named'), weightValueNode);
 					}
 
 					return;
 				}
 
 				if (
-					!keywordSets.fontWeightKeywords.has(weightValue.toLowerCase()) &&
-					weightValue.toLowerCase() !== NORMAL_KEYWORD
+					!keywordSets.fontWeightKeywords.has(lowerWeightValue) &&
+					lowerWeightValue !== NORMAL_KEYWORD
 				) {
-					return complain(messages.invalidNamed(weightValue));
+					return complain(messages.invalidNamed(weightValue), weightValueNode);
 				}
 			}
 
 			/**
 			 * @param {string} message
+			 * @param {import('postcss-value-parser').Node} [valueNode]
 			 */
-			function complain(message) {
+			function complain(message, valueNode) {
+				const index = declarationValueIndex(decl) + (valueNode ? valueNode.sourceIndex : 0);
+
 				report({
 					ruleName,
 					result,
 					message,
 					node: decl,
-					index: declarationValueIndex(decl) + weightValueOffset,
+					index,
 				});
 			}
 		}
 	};
 };
 
+/**
+ * @param {string} value
+ * @returns {import('postcss-value-parser').Node[]}
+ */
+function getWeightValueNodes(value) {
```

Comment:
We generally use `find*` when looking through sub ASTs for things. I'm not sure where the convention started, but shall we follow it here to be consistent, i.e. `findFontWeights(value)`?

Reply:
Thank you. Fixed via 9bb54e1b. 👍🏼